### PR TITLE
Earn: update copy and icon on the Payments splash page in Earn.

### PR DESCRIPTION
**Pull request by donlair**
This PR updates the copy and icon used on the dedicated Payments page that is shown when who has not yet connected to Stripe clicks the Payments card link in the Earn section of Calypso. Context for the changes is at pbMlHh-i6-p2.

Here are screenshots showing the changes:

Before:
Screen Shot 2020-06-23 at 9 24 00 PM

After:
Screen Shot 2020-06-23 at 3 39 25 PM

#### Testing instructions

* Checkout this branch (git checkout update/earn-payments-splash-page-copy-edits)
* Start calypso (yarn start)
* Navigate to the Earn page using an account with a paid plan that has not connected to Stripe (/earn)
* Click the recurring payments card to load the splash page (/earn/payments/)
* Click the CTA button and confirm that the process to connect to Stripe is started.

### CODE REVIEW

File: client/my-sites/earn/memberships/index.jsx [hunk 1]
```diff
@@ -474,41 +479,62 @@ class MembershipsSection extends Component {
...
+							{ this.props.translate( 'You select which premium content requires payment.' ) }
 						</div>
 						<div>
 							<Gridicon size={ 18 } icon="checkmark" />
-							{ this.props.translate( 'Easily manage subscribers' ) }
+							{ this.props.translate( 'Send paid newsletters with premium content.' ) }
+						</div>
+						<div>
+							<Gridicon size={ 18 } icon="checkmark" />
+							{ this.props.translate( 'Collect payments in 135 countries.' ) }
```
Comment (a8ci18n) [inline client/my-sites/earn/memberships/index.jsx:528]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **20** times:
`translate( 'Collect payments in 135 countries' )` *ES Score: 14*

File: client/my-sites/earn/memberships/index.jsx [hunk 2]
```diff
@@ -474,41 +479,62 @@ class MembershipsSection extends Component {
...
+						</div>
+						<div>
+							<Gridicon size={ 18 } icon="checkmark" />
+							{ this.props.translate( 'Easily manage subscribers.' ) }
```
Comment (a8ci18n) [inline client/my-sites/earn/memberships/index.jsx:532]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Easily manage subscribers' )` *ES Score: 11*

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3924242

Thank you @donlair for including a screenshot in the description! This is really helpful for our translators.

**Comment by donlair** [comment]
@danhauk could you take a look at the update? I added a Card component around the main content to get it closer to your updated comps. Does that also address your concerns around the heading / content alignment? I've also adjusted the spacing  and wrapping on the list.

Screen Shot 2020-06-24 at 3 42 37 PM

File: client/my-sites/earn/memberships/index.jsx [hunk 3]
```diff
@@ -471,46 +476,66 @@ class MembershipsSection extends Component {
...
-							{ this.props.translate( 'Easily manage subscribers' ) }
+			<Card>
+				<div className="memberships__onboarding-wrapper">
+					<div className="memberships__onboarding-column-info">
+						<div className="memberships__onboarding-header">
+							{ this.props.translate( 'Accept payments on your website.' ) }
 						</div>
+						<p className="memberships__onboarding-paragraph">
+							{ this.props.translate(
+								'WordPress.com Payments makes it easy to sell physical and digital goods, accept donations, charge for in-person services, build subscription newsletters, and more.'
```
Comment (a8ci18n) [inline client/my-sites/earn/memberships/index.jsx:487]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'WordPress.com Payments makes it easy to sell physical and digital goods, accept donations, charge for in-person services, build subscription newsletters, and more. One-time, monthly, and yearly credit and debit card payment options are supported.' )` *ES Score: 8*

File: client/my-sites/earn/memberships/index.jsx [hunk 4]
```diff
@@ -471,46 +476,66 @@ class MembershipsSection extends Component {
...
+									components: {
+										link: <a href="/support/recurring-payments-button/" />,
+									},
+								}
+							) }
+						</p>
+						<p className="memberships__onboarding-paragraph">{ cta }</p>
+						<p className="memberships__onboarding-paragraph">
+							{ this.props.translate(
+								'Payments are securely processed by Stripe, a payment partner for all credit and debit card payments.'
```
Comment (a8ci18n) [inline client/my-sites/earn/memberships/index.jsx:503]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'Payments are safely, securely, and seamlessly processed by Stripe, a payment partner for all credit and debit card payments. Stripe provides easy access to your money and takes just minutes to set up.' )` *ES Score: 6*

**Comment by donlair** [comment]
Here's an updated screenshot of the current layout in this PR:
Screen Shot 2020-06-26 at 1 23 41 PM

File: client/my-sites/earn/memberships/index.jsx [hunk 5]
```diff
@@ -498,7 +445,7 @@ class MembershipsSection extends Component {
 						) }
 					>
 						<NoticeAction href={ `/earn/payments-plans/${ this.props.siteSlug }` } icon="create">
-							{ this.props.translate( 'Add a payment plan' ) }
+							{ this.props.translate( 'Add a Payment Plan' ) }
```
Comment (a8ci18n) [inline client/my-sites/earn/memberships/index.jsx:448]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3924242).

**Comment by danhauk** [comment]
@donlair Thanks for working to bring it closer to the proposed design. I think it's looking good. I pushed up a couple commits to tweak the styles, add mobile styles, and use some typography variables we have available.

Here's some screenshots:

image

mobile screenshot

**Comment by donlair** [comment]
@danhauk I rebased the update and your changes look good to me. Can you mark this as reviewed? I'll deploy the changes once we get a green light that the name change updates are stable.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3924242).

**Pull request merged**